source/blog/2016-08-29-this-week-in-miq-speedups-chargebacks.html.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Last Week in ManageIQ: Speedups, Chargebacks, Toolbars, and more!"
+date: 2016-08-29
+author: jntullo
```

Comment:
You may also need to add yourself to [data/authors.yml](https://github.com/ManageIQ/manageiq.org/blob/master/data/authors.yml)